docs/sources/reference/components/discovery/discovery.process.md

```diff
@@ -157,6 +159,7 @@ discovery.process "all" {
     commandline = true
     username = true
     uid = true
+    cgroup_path = true
     container_id = true
   }
 }
```

Comment:
A separate example showing how to parse the cgroup in a `discovery.relabel` would be nice.

Not fully understand your usecase, but it looks like you are doing something with qemu vms, maybe relevant/helpful for someone else.

Reply:
Yes, I agree. It would be very useful to have an example.

Well, our use case is to profile HPC batch jobs. We can retrieve job IDs using the `cgroup` name and use this job ID as `service_name` to aggregate the profiles of all processes running under a job.